content/en/apps/features/muting/_index.md

```diff
@@ -0,0 +1,63 @@
+
+### How it is done
+
+To mute a person or a family, navigate to their profile, open the action window, and select the “Mute” form. The form can also be accessed from the Reports tab by choosing a contact to mute from a dropdown list. 
+
+<div class="container">
+  <div class="row">
+{{< figure src="muted_person.png" link="muted_person.png" class="col-9 col-lg-6" >}}
+{{< figure src="muting-family-form.png" link="muting-family-form.png" class="col-6 col-lg-4" >}}
+  </div>
+</div>
+
+Some changes are observed in the UI when a family or a person is muted to make the muted status clear and easily recognizable. The person's or family's icon turns grey  and a status of “Muted” displays on the second line below the person's or family's name. Any actions that were previously available on the family or person remain available on the profile.
+
+{{< figure src="muted-family-appearance-on-list.png" link="muted-family-appearance-on-list.png" class="col-3 col-lg-6" >}}
+
+For the cases where families are muted, each of the family members in the people card are also styled as muted, (grey icons and “Muted” labels).A user may add a new person to a muted household. If they do, and choose to leave the family muted, the new person will be automatically muted once created. When a family is unmuted, all individuals in the family will be unmuted at the same time. 
+
+Unmuting an individual person in a muted family works essentially the same way, the action unmutes the entire family because unmuted individuals  are not allowed underneath a muted household.
+
+### Configuration
+
+Through [configuration]({{< relref "apps/reference/app-settings/transitions" >}}), you can specify which form mutes and unmutes, form availability, form questions and options, manager verification and labels used for display and warning messages.
+
+### Muting vs Death Reporting
+
+A project may support both death reporting and muting - they are not mutually exclusive. Death reporting moves the deceased person to a different part of the family members list and does not allow actions. Muting keeps the person in the family members list, allows actions and disables schedules.
+
+|   Death reporting                                                                                        | Muting                                                                                         |
+| -------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------- |
+| <p></p><ul><li>Permanent state</li></ul>                                                                 | <ul><li>Temporary state</li></ul>                                                              |
```

Comment:
The bullet points do not line up properly for all rows except the last one. You can see that the death reporting bullet points are a bit lower than the ones for Muting. 
 
<img width="840" alt="Screenshot 2023-01-13 at 11 22 21 PM" src="https://user-images.githubusercontent.com/10096985/212450439-ecc21c1c-21da-452d-bfbb-927322a475a0.png">

Reply:
Thanks for noting this, I have corrected the alignment